.editorconfig

```diff
@@ -699,6 +699,7 @@ csharp_style_namespace_declarations = block_scoped:silent
 csharp_style_prefer_method_group_conversion = true:silent
 csharp_style_prefer_top_level_statements = true:silent
 csharp_style_prefer_primary_constructors = true:suggestion
+dotnet_diagnostic.RCS1194.severity = none
```

Comment:
What's the wisdom on not doing the full constructor set for custom exceptions? Is that not recommended practice, anymore? In my mind, I always thought the biggest reason wasn't just for consistency, but that it has something to do with serialization of exceptions, which is of course pretty important for systems that, say, like building crash reports.

Reply:
It is saying that I should implement several different constructors because it's "standard", when the reality is it is created for internal use by dynamic data, in which case I am in total control of which constructors are required, The recommendation is dumb because it is not intended for public use, and if some decides to throw it outside the lib, that is not my concern.